src/components/draw/feature-measurements.vue

```diff
@@ -27,21 +31,36 @@ const feature = ref<DrawnFeature | undefined>(inject('feature'))
 const featureType = ref<string>(feature.value?.featureType || '')
 const featureGeometry = ref<Geometry | undefined>(feature.value?.getGeometry())
 
-const featLength = computed(() =>
-  featureGeometry.value &&
-  ['drawnLine', 'drawnCircle', 'drawnPolygon'].includes(featureType.value)
-    ? getFormattedLength(featureGeometry.value as Geometry, mapProjection)
-    : undefined
-)
-const featArea = computed(() =>
-  featureGeometry.value &&
-  ['drawnPolygon', 'drawnCircle'].includes(featureType.value)
-    ? getFormattedArea(featureGeometry.value as Polygon)
+const featLength = computed(() => {
+  if (featureGeometry.value) {
+    if (['drawnLine', 'drawnPolygon'].includes(featureType.value)) {
+      return getLength(featureGeometry.value as Geometry, mapProjection)
+    } else if (featureType.value === 'drawnCircle') {
+      return getCircleLength(featureGeometry.value as Circle, mapProjection)
+    } else {
+      return undefined
```

Comment:
Is this one last 'else' really useful? l.44 does the same.

Reply:
Oh right, thanks for the catch!